### PR TITLE
Tidy up the JS

### PR DESCRIPTION
**Pull request by friendlyanon**
Some stylish changes. Trying to chip in to make the best even better.

### CODE REVIEW

File: src/dist/js/jquery-2.2.4.min.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 /* NO, really, NOBODY NEEDS TO INCLUDE A FUCKING FRAMEWORK TO MAKE A COOL WEBSITE. */
-console.log('Don\'t worry, no jQuery was used in this website.');
-var $ = function(){this.fancyRainbow = function(){return this;};this.html = function(){return this;};return this;};
\ No newline at end of file
+console.log("Don't worry, no jQuery was used in this website.");
+var _=function(){return $},$=function(){$.fancyRainbow=_;$.html=_;return $}
```
Comment (denysvitali) [inline src/dist/js/jquery-2.2.4.min.js]: This change is going to [break this](https://github.com/denysvitali/thebestmotherfuckingwebsite/blob/master/src/pug/scriptjoke.html)  
I added this line to fake a jQuery function call

**Comment by denysvitali** [comment]
Except for the underscore change, everything else LGTM. Thanks for your contribution!

File: src/dist/js/jquery-2.2.4.min.js (hunk 1 above)
Comment (friendlyanon) [reply src/dist/js/jquery-2.2.4.min.js]: Indeed, I was worried that there could be stack related issues, but turns out I was just being paranoid. Pushed a new commit to fix this, so there should only be one exported symbol to the global scope like before :wink:

**Comment by denysvitali** [comment]
Thank you 🦖

File: src/dist/js/main.fucking.js [hunk 2]
```diff
@@ -1,45 +1,42 @@
...
   r.style.color = color;
-  setTimeout(function(){doThatFuckingColorThing();}, 1000/25);
+  setTimeout(doThatFuckingColorThing, rainbowTiming);
 }
 
-function someControl(id, textArr, className){
-  // You see? No fucking jQuery needed, check http://jsperf.com/getelementbyid-vs-jquery-id/44
-  var acbox = document.getElementById(id);
-  var clickEvent = ('ontouchstart' in window ? 'touchend' : 'click');
-  var el = document.getElementsByTagName('html')[0];
```
Comment (denysvitali) [inline src/dist/js/main.fucking.js]: @friendlyanon: RIP `el` :sob:

**Pull request merged**